components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -516,59 +441,77 @@
       this._editorLauncher.observe('click', function(event) {
         event.stop();
         event.findElement().blur();
-        _this._launchEditor();
+        _this._populateEditorData();
       });
       // ---------------------------------------------------------------------
       // Initialization done
       // ---------------------------------------------------------------------
     },
+
+    // Access levels and visibility options are fetched from patient from hidden inputs generated by velocity
     _initAccessLevels : function() {
       var levels = {'access-level' : [], 'visibility' : []};
       this._element.select('input[name="access-level"], input[name="visibility"]').each(function (item) {
-        levels[item.name].push({id : item.value, name : item.alt, title : item.title, icon: item.next('span.fa') || ''});
+        levels[item.name].push({id : item.value, name : item.alt, title : item.title, icon : item.next('span.fa') || ''});
       });
       this.ACCESS_LEVELS = levels['access-level'];
       this.VISIBILITY = levels['visibility'];
     },
+
     _initManagerUI : function() {
-      this._container = new Element('form', {'id' : 'access-levels-manager', 'class' : 'xform', 'method' : 'post', 'action' : this._serviceURL});
-      //this._element.insert({'after' : this._container});
+      this._container = new Element('form', {'id' : 'access-levels-manager', 'class' : 'xform'});
       this._container.insert(this._generateSection('manage-ownership', "$services.localization.render('phenotips.patientAccessRightsManagement.modifyOwnershipHeader')", "$services.localization.render('phenotips.patientAccessRightsManagement.modifyOwnershipBody')"))
                      .insert(this._generateSection('manage-visibility', "$services.localization.render('phenotips.patientAccessRightsManagement.modifyVisibilityHeader')", "$services.localization.render('phenotips.patientAccessRightsManagement.modifyVisibilityBody')"))
                      .insert(this._generateSection('manage-collaborators', "$services.localization.render('phenotips.patientAccessRightsManagement.modifyCollaboratorsHeader')", "$services.localization.render('phenotips.patientAccessRightsManagement.modifyCollaboratorsBody')"))
+                     .insert(this._generateSection('manage-journals', 'Journals', 'If this case is being submitted to one of the following journals, please check the appropriate box. You give the journal permission to access this record, and to make this record public if the case is published.'))
```

Comment:
Sure, was too deep into Phonomecentral lately and tested stuff there assuming code is same, which is surely not a right way. But at least one positive thing is now I know about this journals addon. Do you want me to fix it as well in different PR?